source/slang/slang-compiler.cpp

```diff
@@ -971,6 +1060,14 @@ namespace Slang
         //
         List<D3D_SHADER_MACRO> dxMacrosStorage;
         D3D_SHADER_MACRO const* dxMacros = nullptr;
+
+        IncludeHandlerImpl includeHandler;
+        includeHandler.linkage = linkage;
+        includeHandler.searchDirectories = &linkage->searchDirectories;
+
+        FxcIncludeHandler fxcIncludeHandlerStorage;
```

Comment:
Should this be FXC or Fxc? I mean I prefer Fxc - it scans better, and perhaps subconsciously you do too ;)